### DISCUSSION TITLE
Components have no styles but have tailwindcss classes

### DISCUSSION THREAD

**Discussion by black-purple** [huntabyte/shadcn-svelte#904] [Help]
After going through the config in the official docs and importing the components in my svelte page I don't see any styling, just a plain button or a plain input not styling at all and I think the problem is with either postcss or tailwind cuz when I try to style other tags with tailwind it doesn't work at all as if tailwind is not there.

**Comment by AleksanderKGH** [comment]
I'm wondering if you still have this issue...
I realized I didn't have tailwind installed (after some troubleshooting of course... 🤦‍♂️)

**Comment by chathura-de-silva** [comment]
I am facing the same issue.

**Comment by philmas** [comment]
Same here. It seems broken.

**Reply by philmas** [reply]
> Same here. It seems broken.

I managed to fix it by adding the following to the app.css
```
@source '../src/**/*.{html,js,svelte,ts}';
@source '../src/lib/components/**/*.{svelte,ts}';
```

**Reply by MeowyCat1** [reply]
Doesn't seem to work for me...

**Reply by MeowyCat1** [reply]
Actually, it did work when I added @source "../lib/components/ui";

**Reply by MeowyCat1** [reply]
Surely this should be in the docs... I might create an issue about this.

**Reply by MeowyCat1** [reply]
https://github.com/huntabyte/shadcn-svelte/issues/2459

Here it is ⬆️

**Reply by Zer0F8th** [reply]
@MeowyCat1 I saw the same issues as you and I followed the docs to the letter, adding the following to `layout.css` worked for me as well:

```css
@source "../lib/components/ui";
```

**Reply by SzmiSmi** [reply]
@Zer0F8th's solution worked for me. Alternatively, tailwind also started working on my shadcn components when I moved my own component living at '/lib' to inside '/lib/components/ui'.

**Comment by MeowyCat1** [comment]
If I follow the sveltekit tutorial and add a button it doesn't have styling at all

**Comment by black-purple** [comment]
I don't remember how I solved it, I think when I cleared the root layout file I deleted the css import. I tried everything then even uninstalling everything and doing a clean tailwind install, and made sure the css file was imported in the root layout.
